src/Symfony/Component/DependencyInjection/Compiler/ResolveReferencesToAliasesPass.php

```diff
@@ -27,6 +27,7 @@ class ResolveReferencesToAliasesPass extends AbstractRecursivePass
      */
     public function process(ContainerBuilder $container)
     {
+        $this->enableExpressionProcessing();
```

Comment:
Are you sure about that ? Replacing a reference inside the expression won't change the expression anyway. It will keep using the alias.